docs/src/pages/configurations/typescript-config/index.md

```diff
@@ -78,8 +78,16 @@ Change `config.ts` inside the Storybook config directory (by default, it’s `.s
 ```js
 // automatically import all files ending in *.stories.js
 const req = require.context('../stories', true, /.stories.tsx$/);
+
+function loadStories() {
+  req.keys().forEach(req);
+}
+
+configure(loadStories, module);
 ```
 
+If you have your `/stories` directory in `src`, change the above `../stories` to `../src/stories`.
```

Comment:
I don't think this line is all that needed. It's kinda implied that the route points to your stories folder.

Reply:
I was on the fence about it, too. Removed the line.